src/main/java/com/mifos/services/data/LoansPayload.java

```diff
@@ -22,6 +22,7 @@
     int clientId;
     String externalId;
     String dateFormat;
+    String timeFormat;
```

Comment:
What is time format ?
Does it will not have date component ?
Are we introducing a new format in Mifos ? If yes, why not dateTimeFormat ?

Reply:
no, it is already existing in mifos. 
any thing which is to be read as datetime will have an accompanying timeformat to specify format